source/Public/Connect-FabricAccount.ps1

```diff
@@ -12,33 +12,73 @@ function Connect-FabricAccount {
     The TenantId of the Azure Active Directory tenant you want to connect to
     and in which your Fabric Capacity is.
 
+.PARAMETER ServicePrincipalId
+    The Client ID (AppId) of the service principal used for authentication.
+
+.PARAMETER ServicePrincipalSecret
+    The **secure string** representing the service principal secret. Use Read-Host -AsSecureString or other secure entry.
```

Comment:
Could we recommend reading from a key vault here please to encourage good practice rather than a human entering secrets!! 😉